package.json

```diff
@@ -18,11 +18,17 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
```

Comment:
Yup! This is something I'll include in that follow-up PR with a stricter set of rules!

(unfortunately, we can't do it for `package.json` since JSON itself doesn't *allow* trailing commas after objects, but I can enforce it for all js files)